### PR TITLE
feat: added API tests for `GET` `/numbers` and `GET` `/number-types`

### PR DESCRIPTION
**Pull request by avaitonis**
API tests for GET /numbers
API tests for POST /numbers
API tests for GET /number-types
Refactor GET /numbers to improve query params validation
Refactor POST /numbers to improve results validation, added try catch
Refactor /test/api.js to pass eslint validations
Change eslint command. Exclusion folders are set in eslint config
Change database connections to use UTC time. This DB saves data using UTC, API results also has time in UTC.

### CODE REVIEW

File: test/numbers/numbers.api-test.ts [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        createdBy: 'Jest',
+        requestingSystem: 'Jest 4 - Covenant',
+      },
+    ];
+    const { status, body } = await api.post(payload).to('/numbers');
+    expect(status).toEqual(201);
+    expect(body).toHaveLength(4);
+
+    /* eslint-disable security/detect-object-injection */
+    for (let i = 0; i++; i < 4) {
```
Comment (IgnatG) [inline test/numbers/numbers.api-test.ts:215]: I wonder if `test.each()` will be more appropriate here?
> https://jestjs.io/docs/api#testeachtablename-fn-timeout

**Review by IgnatG — APPROVED** [review]
Minor comments - but it's looking really good overall!

File: src/modules/numbers/numbers.service.ts [hunk 2]
```diff
@@ -28,11 +30,25 @@ export class NumbersService {
...
   async findOne(type: number, ukefIdString: string): Promise<UkefId> {
-    const dbNumber = await this.numberRepository.query('USP_STP_GET_AUTONUMBER @0, @1', [type, ukefIdString]);
-    if (!dbNumber[0]) {
-      throw new NotFoundException('UKEF ID is not found');
+    try {
+      const dbNumber = await this.numberRepository.query('USP_STP_GET_AUTONUMBER @0, @1', [type, ukefIdString]);
+      if (!dbNumber[0]) {
+        throw new NotFoundException('UKEF ID is not found');
+      }
+      if (dbNumber[0]?.ERR && dbNumber[0].ERR === 'INVALID NUMBER TYPE') {
```
[the remark this answers is not in the document]
Comment (avaitonis) [reply src/modules/numbers/numbers.service.ts:38]: Will change in next PR

File: test/numbers/numbers.api-test.ts (hunk 1 above)
Comment (avaitonis) [reply test/numbers/numbers.api-test.ts:215]: Will change in next PR

**Pull request merged**